src/course-home/outline-tab/Section.jsx

```diff
@@ -41,9 +44,14 @@ const Section = ({
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const activeSequenceId = useSelector(state => state.courseware.sequenceId);
```

Comment:
[question]: Can we move this into a file with selectors?

Reply:
All changes in `src/course-home/outline-tab` will be reverted in next PR: https://github.com/raccoongang/frontend-app-learning/pull/17